server/graphql_schemas/event/schema.py

```diff
@@ -357,12 +359,46 @@ class SlackChannelsList(graphene.ObjectType):
     class Meta:
         interfaces = (relay.Node,)
 
+
+class ShareEvent(relay.ClientIDMutation):
+    class Input:
+        event_id = graphene.ID()
+        channel_id = graphene.ID()
+
+    event = graphene.Field(EventNode)
+
+    @classmethod
+    def mutate_and_get_payload(cls, root, info, **input):
+        event_id = input.get('event_id')
```

Comment:
The `event_id` here will be a global id sent from the client. We should have convert to input to database id using `from_global_id()` utility.

Reply:
Noted